starter-code/iteration-3/main.js

```diff
@@ -1 +1,49 @@
-"use strict";
+class TodoItem {
+    constructor(title) {
+        this.title = title;
+        this.isDone = false;
+        this.updatedAt = new Date();
+    }
+    done() {
```

Comment:
toggleStatus() {
  this.isDone = !this.isDone;
  this.updatedAt = new Date();
}